diff -Naur sudoku-1.0/config_sdk sudoku-1.0-new/config_sdk
--- sudoku-1.0/config_sdk	2011-08-25 13:56:45.000000000 +0800
+++ sudoku-1.0-new/config_sdk	2011-08-26 16:27:51.000000000 +0800
@@ -1,2 +1,2 @@
 This is configuration of sudoku.
-Sudoku_UI=OFF
+Sudoku_UI=ON
diff -Naur sudoku-1.0/releasenote sudoku-1.0-new/releasenote
--- sudoku-1.0/releasenote	2011-08-25 13:57:13.000000000 +0800
+++ sudoku-1.0-new/releasenote	2011-08-26 16:30:37.000000000 +0800
@@ -5,3 +5,7 @@
 Basic Makefile
 Edit the warning errors
 ==================================
+
+2011-8-26
+Config flag set to ON.
+==================================
